src-docs/src/views/accessibility/accessibility_example.js

```diff
@@ -75,40 +62,119 @@ export const AccessibilityExample = {
               provides functionality or interactivity) is important enough to
               provide to screen reader users, it should probably be made
               available to all users.&quot;{' '}
-              <EuiLink
-                href="http://webaim.org/techniques/css/invisiblecontent/"
-                external
-              >
+              <EuiLink href="http://webaim.org/techniques/css/invisiblecontent/">
                 Learn more about invisible content
               </EuiLink>
             </p>
           </EuiCallOut>
-        </div>
+          <EuiSpacer />
+          <p>
+            <em>
+              Using a screen reader, verify that there is a second paragraph.
+            </em>
+          </p>
+        </>
       ),
       props: {
         EuiScreenReaderOnly,
       },
-      snippet: screenReaderOnlySnippet,
+      snippet: `<EuiScreenReaderOnly>
+  <!-- visually hidden content -->
+</EuiScreenReaderOnly>`,
       demo: <ScreenReaderOnly />,
     },
     {
-      title: 'Skip link',
       source: [
         {
           type: GuideSectionTypes.JS,
-          code: skipLinkSource,
+          code: screenReaderButtonSource,
+        },
+      ],
+      text: (
+        <>
+          <h3>Within a button</h3>
+          <p>
+            If the screen reader content is <strong>contained within</strong> a
+            focusable element, you should consider adding{' '}
+            <EuiCode language="scss">{'position: relative;'}</EuiCode> to the
+            focusable element. This will fix any screen reader focus rings to
+            stay within the bounds of the focusable element.
```

Comment:
After some thought, I'm on the fence about adding an extra documentation section for this behavior, since as far as I can tell it mostly applies to just Safari + VO.

We might want to evaluate end-user usage and whether it's worth adding this documentation/advisement for a single browser+screenreader combo that WebAim reports in its 2021 survey to be [5% of desktop screen reader users](https://webaim.org/projects/screenreadersurvey9/#browsercombos) (compared to JAWS and NVDA on Windows, which is [90%+](https://webaim.org/projects/screenreadersurvey9/#os)). I think:

1. This is a non-issue for the vast majority screen reader users on desktop
    1. Although I do want to caveat that for mobile screen reader users (50%+ of SR users), iOS Safari + VO is 50% of the market share. However, VO doesn't behave the same way on mobile as it does on desktop and this may be a non-issue there - we won't know for sure until we test on an iOS device. But to that point: IIRC we don't technically support/test EUI or Kibana on mobile devices.
2. This is not an issue for Safari keyboard users, as Safari does not show the off-screen outline hitbox when not using VO
3. This is also purely a visual outline issue - for a tool that is meant for non-visual users. I acknowledge there's a spectrum of vision impairment and it's possible some screen reader users will still see the off-screen outline and may be slightly confused by it, but nevertheless the audio behavior still works perfectly. With that, I'm going to argue that it's not worth it to document a workaround for a problem that maybe half of 5% of our users might affect (and not functionally).

To clarify, I'm 100% on board with adding `position: relative` for our external EUI links since it's a fairly common pattern and a small/easy fix for us, I just think this extra documentation section for it is unnecessary since it's incredibly browser+screen reader specific and extremely low-impact, as above.

Reply:
I just err on giving the consumers full knowledge up front of known issues and how to work around them. If you feel strongly, feel free to remove.